modules/model-context-protocol-tools/proc-resolving-common-problems-with-mcp-clients-and-tools.adoc

```diff
@@ -0,0 +1,43 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="proc-resolving-common-problems-with-mcp-clients-and-tools.adoc_{context}"]
+= Resolving common problems with MCP client and tools
+
+The following procedures guide you through resolving common issues when using a Model Communication Protocol (MCP) client for tool calling.
+
+== Resolving the `Model does not support tool calling` error
+
+This error indicates that the model configured in your MCP client lacks the required functionality to handle tool calls. The error message appears similar to: `Invalid request: model gemma3:27b does not support tool calls`.
+
+.Procedure
+
+. Consult your model documentation to confirm its support for tool calling.
+. If the current model does not support tool calling, change the model that your MCP client uses to a tool-calling compatible model.
+
+== Resolve authentication issues when tools are not found
+
+If your MCP client connects to the server but cannot find deployed tools, an authentication or configuration issue is likely.
+
+.Procedure
+
+. Verify the authentication token.
```

Comment:
Could we add a small section to this step (verify the authentication token).

`Set this token as the bearer token in your MCP client` could we also specify "ensure the authorization header in the configuration specifies `Bearer` before the token. E.g. `Bearer <mcp-token>`"` 

(or something to that effect)